scripts/HFIR_4Circle_Reduction/IntegrateSingePtSubWindow.py

```diff
@@ -73,6 +75,19 @@ def __init__(self, parent=None):
 
         return
 
+    def _promote_widgets(self):
```

Comment:
This is great function that could be made free and added to `scripts/gui_helper.py` in order to reduce duplicated code. I think it would need to take a frame as the argument and return the canvas so the caller can attach connections as necessary.